app/views/admin/settings/index.html.haml

```diff
@@ -101,6 +101,13 @@
                     = I18n.t('admin.members.surname')
                     = f.text_field :last_name, :value => @admin.last_name, :class => 'form-control'
 
+              .form-group
+                .row
+                  .col-md-12
+                    %label{for: ''}
+                    = I18n.t('activerecord.attributes.member.language')
+                    = f.select :language, options_for_select([["Nederlands", :nl], ["Engels", :en]], selected: @user.language)
```

Comment:
Shouldn't those string be translated as well (Dutch, English)?

Reply:
This was something we discussed when implementing the language switch in `static-sticky` as well. Most of the times when switching languages, you would want to see the text in its native language. This is pretty much standard everywhere.

Reply:
In that case, shouldn't it be Nederlands, English?

Reply:
Oh shoot, you're right, will fix that whenever I have time